lib/admin/platforms.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Admin.Platforms do
+  @moduledoc """
+  Generate platform URLs.
+  """
+
+  defp base_host, do: Application.get_env(:admin, :base_host)
+
+  def get_url(:library, item_id) do
```

Comment:
Should you check item_id validity? ie. empty string, nil, etc?
Applies for all functions there.

Reply:
I added tests. 

I can add a guard to the function, so that it does not match if it is not a string. But what should we do if it does not match ? should we provide a default to the library home ? return an empty string ? let it crash when called ?